dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -79,12 +90,58 @@ def self.get_lesson_s3_name(script_level)
     [code, version]
   end
 
-  private def get_fake_openai_evaluations(rubric, understanding_s: 'Extensive Evidence')
-    rubric.learning_goals.map do |learning_goal|
-      {
-        'Key Concept' => learning_goal.learning_goal,
-        'Grade' => understanding_s
-      }
+  private def read_file_from_s3(lesson_s3_name, key_suffix)
+    bucket = 'cdo-ai'
+    key = "teaching_assistant/lessons/#{lesson_s3_name}/#{key_suffix}"
+    s3_client.get_object(bucket: bucket, key: key)[:body].read
+  end
+
+  private def read_examples(lesson_s3_name)
+    bucket = 'cdo-ai'
```

Comment:
This is hard coded in a few places. Could we make the bucket name a constant? We have this similar pattern in a few places: `S3_AI_BUCKET = 'cdo-ai'.freeze`

Reply:
good idea! done